src/components/Footer/IconLink.js

```diff
@@ -1,29 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
-import IconButton from '@material-ui/core/IconButton';
 import Link from '@material-ui/core/Link';
 
-const styles = () => ({
+const styles = theme => ({
 	icon: {
-		width: '30px'
+		width: '30px',
+		margin: theme.spacing(1.5)
 	}
 });
 
-function IconLink({ classes, icon: Icon, link }) {
+function IconLink({ classes, icon: Icon, link, ariaLabel }) {
```

Comment:
Hmm, would it be possible to call this `aria-label` to be consistent?

```suggestion
function IconLink({ classes, icon: Icon, link, 'aria-label': ariaLabel }) {
```